client/engine/engine.go

```diff
@@ -536,16 +525,16 @@ func (e Engine) spawnConsensusChannelIfDirectFundObjective(crankedObjective prot
 }
 
 // getOrCreateObjective retrieves the objective from the store. if the objective does not exist, it creates the objective using the supplied signed state, and stores it in the store
```

Comment:
godoc refers to signed state

Reply:
[38fc864](https://github.com/statechannels/go-nitro/pull/864/commits/38fc864269bc3d91827ebdbf9e203f2312db572a)